.travis.yml

```diff
@@ -2,9 +2,9 @@ sudo: false
 language: python
 python:
   - "2.7"
-  - "3.3"
   - "3.4"
   - "3.5"
+  - "3.6"
```

Comment:
Same as https://github.com/OpenDataServices/flatten-tool/pull/196/files#r207384510

Reply:
This change is now on the master branch, so doesn't appear in the diff.